images/build/Dockerfiles/Dockerfile

```diff
@@ -166,11 +166,19 @@ RUN set -ex \
  && ln -s 5.4.2 /opt/npm/5.4 \
  && ln -s 5.6.0 /opt/npm/5.6 \
  && ln -s 5.6 /opt/npm/5
+RUN set -ex \
+    && mkdir -p /output \
+    && cd /opt \
+    && tar -zcf /output/opt.tar.gz .
 
 FROM main AS final
-COPY --from=intermediate /opt /opt
+COPY --from=intermediate /output /opt
```

Comment:
so this will copy the yan-cache, but nuget contents are still in /var/nuget .. can you please confirm /var/nuget is also getting copied in /output?

Reply:
nvm .. i  saw it's only need in ltsversions image which is there .. so ignore this 👍